tests/unit_tests.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import pytest
+
+from apparun.impact_methods import MethodFullName, MethodShortName
+from apparun.impact_model import ImpactModel, ModelMetadata
+from apparun.impact_tree import ImpactTreeNode
+from apparun.parameters import ImpactModelParams
+from apparun.results import get_result
+
+
+def test_impact_tree_calculation():
+    impact_model = ImpactModel().from_yaml("data/noparam_system.yaml")
+    lcia_scores = impact_model.get_scores()
+    scores = sorted(lcia_scores.scores.items())
+    assert dict(scores) == {'EFV3_CLIMATE_CHANGE': 1.3830513221576728e-06, 'EFV3_ECOTOXICITY_FRESHWATER': 4.365302017373894e-12, 'EFV3_LAND_USE': 0.0}
+
+def test_impact_calculation_result():
```

Comment:
shouldn't the test contain an assertion?

Reply:
I could test that the files are created